setup.py

```diff
@@ -34,7 +34,7 @@
     ],
     'tests': tests_require,
     'sentry': [
-        'raven[flask]>=5.0.0,<5.5',
+        'raven[flask]>=6.0',
```

Comment:
What is the reason for this one? I have a vague recollection that there were some issues with this in the past.

Reply:
It was probably too much to upgrade to `>=6.0`. I actually only needed `>=5.10.2` to set the `logging_exclusions` but all it does is set `logging.getLogger('py.warnings').propagate = False` so we could actually skip upgrading at all and manually set `propagate`.

I also believe there was an incompatibility between Invenio-Logging's Raven version and Flask-Login which seems to have been fixed in `>=5.7.0`.

Is it fine if we change it to `5.10.2,<6` or do you know if there were any issues with those versions?